ecshop/xianzhi-2017-02-82239600/README.md

```diff
@@ -1,30 +1,34 @@
-# ECShop 2.x/3.x SQL注入/任意代码执行漏洞
+# ECShop 2.x/3.x SQL Injection / Remote Code Execution Vulnerability
 
-ECShop是一款B2C独立网店系统，适合企业及个人快速构建个性化网上商店。系统是基于PHP语言及MYSQL数据库构架开发的跨平台开源程序。
+[中文版本(Chinese version)](README.zh-cn.md)
 
-其2017年及以前的版本中，存在一处SQL注入漏洞，通过该漏洞可注入恶意数据，最终导致任意代码执行漏洞。其3.6.0最新版已修复该漏洞，vulhub中使用其2.7.3最新版与3.6.0次新版进行漏洞复现。
+ECShop is a B2C independent shop system for companies and individuals to quickly build personalized online store. This system is a cross-platform open source program based on PHP language and MYSQL database architecture.
 
-参考链接：
+In 2017 and previous versions, there was a SQL injection vulnerability that could inject payload and eventually lead to code execution vulnerabilities. The latest version of 3.6.0 has fixed the vulnerability, and vulhub uses its latest version 2.7.3 and 3.6.0 non-latest version versions to reproduce the vulnerability.
+
+Reference link:
 
 - http://ringk3y.com/2018/08/31/ecshop2-x代码执行/
 
-## 环境搭建
+## Environment construction
```

Comment:
I feel `Environment Setup` is more idiomatic

Reply:
Also if we update here we need to update everywhere as well

Reply:
sure, we need a common description.